### PR TITLE
Add adobe fonts for #4589

### PR DESCRIPTION
**Pull request by ThatGeekyWeeb**
PR adds adobe fonts for #4589, as well as `encodings`.
> 75dpi fonts will be added in later PR

### CODE REVIEW

File: packages/encodings.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  description 'X11 Font Index Generator'
+  homepage 'https://xorg.freedesktop.org'
+  version '1.0.5'
+  compatibility 'all'
+  source_url 'https://www.x.org/releases/individual/font/encodings-1.0.5.tar.bz2'
+  source_sha256 'bd96e16143a044b19e87f217cf6a3763a70c561d1076aad6f6d862ec41774a31'
+
+  def self.build
+      system "./configure #{CREW_OPTIONS} --with-fontrootdir=#{CREW_PREFIX}/share/fonts"
+      system "sed -e 's|^\(encodings_DATA = $(DATA_FILES)\).*|\1|' -i Makefile" # Found in xbps-src
```
Comment (uberhacker) [inline packages/encodings.rb:13]: Can we put this line in a `def self.patch` section?

**Comment by uberhacker** [comment]
Thank you for the new packages @ThatGeekyWeeb!

File: packages/encodings.rb (hunk 1 above)
Comment (ThatGeekyWeeb) [reply packages/encodings.rb:13]: does `self.patch` run before or after `self.build` though? @uberhacker

Comment (uberhacker) [reply packages/encodings.rb:13]: Yes, the order of operations is below:
```
  # Function to perform patch operations prior to build from source.                                                                                                                          
  def self.patch                                                                                                                                                                              
                                                                                                                                                                                              
  end                                                                                                                                                                                         
                                                                                                                                                                                              
  # Function to perform pre-build operations prior to build from source.                                                                                                                      
  def self.prebuild                                                                                                                                                                           
                                                                                                                                                                                              
  end                                                                                                                                                                                         
                                                                                                                                                                                              
  # Function to perform build from source.                                                                                                                                                    
  def self.build                                                                                                                                                                              
                                                                                                                                                                                              
  end                                                                                                                                                                                         
```
Might be good to add this info in Manual.md also. :)

Comment (ThatGeekyWeeb) [reply packages/encodings.rb:13]: The `sed` command needs to be done in the make file, so `./configure` has to run before it. @uberhacker

Comment (uberhacker) [reply packages/encodings.rb:13]: No Makefile exists before the `./configure` command is run?

Comment (ThatGeekyWeeb) [reply packages/encodings.rb:13]: `./configure` generates the `Makefile`, thats the purpose of automake... @uberhacker

**Pull request merged**